openlibrary/plugins/upstream/addbook.py

```diff
@@ -677,6 +692,9 @@ def POST(self, key):
 
         add = (edition.revision == 1 and work and work.revision == 1 and work.edition_count == 1)
 
+        # Invalidate the cache; temporary fix to #
```

Comment:
Missing issue number

Reply:
XP Nice catch!

Reply:
Actually I'll just delete this; I mention it in the method's docstring.
```suggestion
```